test/spec.test.js

```diff
@@ -35,17 +35,15 @@ const blackList = new Set([
   'manifest-rdf#test170',
   'manifest-rdf#test171',
   'manifest-rdf#test183',
-  'manifest-rdf#test187',
-  'manifest-rdf#test188',
-  'manifest-rdf#test189',
-  'manifest-rdf#test190',
+  'manifest-rdf#test188', // date/time
```

Comment:
is it worth mentioning that a copy of the same tests is available in the support folder?

Reply:
Yea, good idea